htcanalyze/main.py

```diff
@@ -91,7 +97,7 @@ def setup_logging_tool(log_file=None, verbose_mode=False):
         both modes are compatible together
     :return:
     """
-    # disable the loggeing tool by default
+    # the logging tool is by default disabled
```

Comment:
Repeat question as above: why?

Reply:
I think `# Disable logging for [reason]` fits better. Brevity, has the reason in it. For example, `# Enable logging when testing`.

If the 'why' is indeed to disable logging for production, but enable it for testing, I recommend against this. Your testing log output should be routed to `logging.debug()`. When you move to production, you simply change the logging level to something higher than debug, such as warning. That way all debug messages will be ignored, but warnings, errors, and critical log messages will still show up.